server/helpers/has-rich-article.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+module.exports = exports = (prodList= []) => {
+	return (prodList.length >= 1) ? prodList.some(product => product.code === 'S2'): false;
+};
```

Comment:
My 2p, there's no need to put this one liner in a separated file, it's only used once. 

If it needs to be in separated file, then it needs test

Reply:
Tests added. Done as a helper as I was thining this might be used on different routes as an when but now it's in the middleware perhaps not.